rpcs3/emucore.vcxproj.filters

```diff
@@ -1806,7 +1806,7 @@
     </ClInclude>
     <ClInclude Include="Emu\SysCalls\Modules\sceNpUtil.h">
       <Filter>Emu\SysCalls\Modules</Filter>
-    </ClInclude>
+    </ClInclude>
```

Comment:
If it was hard, I wouldn't argue, but it's about 10 sec to discard changes and just redo them during rebasing.